text/src/autogluon/text/automm/data/process_image.py

```diff
@@ -83,6 +84,10 @@ def __init__(
                 Use an image with zero pixels.
         requires_column_info
             Whether to require feature column information in dataloader.
+        trivial_augment_maxscale
```

Comment:
If trivialaugment is not sensitive to `trivial_augment_maxscale `, then we may not need this hyper-parameter.